### DISCUSSION TITLE
feature: add linebreak when sending long msg outputted by cmd to split

### DISCUSSION THREAD

**Discussion by Subjective** [folke/noice.nvim#548] [Q&A] [answered]
### Did you check the docs?

- [X] I have read all the noice.nvim docs

### Is your feature request related to a problem? Please describe.

When executing a command that echoes a long message, the message is sent to a split as I would expect. However, the formatting is a bit off as the first line of output is on the same line as the command as shown in the below screenshots.

### Describe the solution you'd like

Add a blankline after the line displaying the command that outputted the long message.

### Describe alternatives you've considered

N/A

### Additional context

Noice:
<img width="1751" alt="image" src="https://github.com/folke/noice.nvim/assets/56745535/66640928-1372-4f59-8225-422fb8fc4fb8">
Noice Disabled/Default View:
<img width="1748" alt="image" src="https://github.com/folke/noice.nvim/assets/56745535/998a4d5c-5851-450e-8f9f-643971b0d8e7">

**Comment by folke** [comment]
You can change the format. Look to the docs/code to see how

**Comment by Subjective** [comment, marked as answer]
Thanks, I was able to get the formatting I wanted:
```lua
views = {
  cmdline_output = {
    format = {
      "{level} ",
      "{date} ",
      "{event}",
      { "{kind}", before = { ".", hl_group = "NoiceFormatKind" } },
      " ",
      "{title} ",
      "{cmdline} ",
      "\n",
      "{message}",
    },
    view = "split",
  },
}, ---@see section on views
```
